src/app/story/directives/hero.component.ts

```diff
@@ -46,6 +47,12 @@ import { StoryModel } from '../../shared';
              [working]="story.isPublishing" (click)="togglePublish()" orange=1>
             {{story.publishedAt ? 'Unpublish' : 'Publish'}}
           </publish-button>
+          <p *ngIf="!story.changed() && !story.publishedAt">
+            Will be queued up for publishing {{story.releasedAt ? ' at ' + formatDate(story.releasedAt) : 'immediately'}}
```

Comment:
Or another alternative: use the current `Last saved at` status text to indicate whatever hints we need about published status.